packages/components/pages/f-self-exclusion/src/store/selfExclusion.module.js

```diff
@@ -10,8 +10,8 @@ export default {
     actions: {
         async getExclusions ({ commit }, { api, authToken }) {
```

Comment:
Couldn't see any tests for this component. Worth adding tests (in a different pr is fine by me).

Reply:
We did not consider adding tests to the component. If required we can add in a different PR.

Reply:
There's some examples you can look at in the other components which you can follow so should help. Give me a shout if you need a hand with these.